contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -792,6 +792,12 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 		}
 	}
 
+	void enterKill_statement(TSqlParser::Kill_statementContext *ctx) override {
+		if (in_create_or_alter_function){
```

Comment:
What about a procedure? Is that a valid case?

Reply:
Yes, it's valid and there is a test case for that. As well as for the invalid case in a function.